core_impl.c

```diff
@@ -400,16 +400,17 @@ bool core_unload(void)
    return true;
 }
 
-
 bool core_unload_game(void)
 {
    video_driver_free_hw_context();
 
    video_driver_set_cached_frame_ptr(NULL);
 
-   current_core.retro_unload_game();
-
-   current_core.game_loaded = false;
+   if (current_core.game_loaded)
```

Comment:
When is 'game_loaded' true? Only when a ROM is loaded?

What if a libretro core does certain core deinitialization in its 'unload_game' callback that has nothing to do with the ROM? Say a contentless core like 2048 or some other core that doesn't require a ROM - do we have any examples of that, and are there then issues with that core?

Reply:
For a contentless game, `current_core.retro_load_game(NULL)` is called, then `game_loaded` is true.

Reply:
OK, that addresses my concern then.